### DISCUSSION TITLE
[1517290] 战地风云™ 2042 | 本体

### DISCUSSION THREAD

**Discussion by 1nanqi1** [pjy612/SteamManifestCache#516] [Share]
`appid:1517290`
 keys:
 ```
1517291;d7387ee0dddd7c974afd4dbea141d732c7c5735adfaad640ab2f77f09322cdd9
1517299;cb7d068c172a90d88b96d96ddf6b69a441e071002e07245731a92d6173799e46
 ```
 code;04A70BC9DCC875CE
 email;1990668148@qq.com
[2042.zip](https://github.com/pjy612/SteamManifestCache/files/14387200/2042.zip)

话说有没有内测群，麻烦拉我下

**Comment by Malik506-dev** [comment]
will online run ?
